src/main/java/racingcar/service/RacingCarService.java

```diff
@@ -0,0 +1,57 @@
+package racingcar.service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import racingcar.model.Car;
+import racingcar.utlis.Convertor;
+import racingcar.validator.NameValidator;
+
+public class RacingCarService {
+
+	List<Car> cars = new ArrayList<>();
+
+	public void playRound() {
+		cars.forEach(Car::decideMove);
+	}
+
+	public void saveCars(String carsNames) throws Exception {
+		List<String> carNameList = new ArrayList<>(Arrays.asList(Convertor.separateNamesByDelimiter(carsNames)));
```

Comment:
ArrayList를 한번 더 생성한 이유가 있을까요?
아래처럼 하면 되지 않을까요!
```suggestion
		List<String> carNameList = Arrays.asList(Convertor.separateNamesByDelimiter(carsNames));
```